.taskcluster.yml

```diff
@@ -57,3 +57,70 @@ tasks:
           description: rust-code-analysis lint and test
           owner: cdenizet@mozilla.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
+
+      - $if: 'tasks_for == "github-push" && head_branch[:10] == "refs/tags/"'
+        then:
+          taskId: {$eval: as_slugid("build_release")}
+          dependencies:
+            - {$eval: as_slugid("lint_test_task")}
+          created: {$fromNow: ''}
+          deadline: {$fromNow: '2 hour'}
+          provisionerId: proj-relman
+          workerType: ci
+          payload:
+            maxRunTime: 3600
+            image: "rust:buster"
+            env:
+              PROJECT_NAME: rust-code-analysis
+            command:
+              - "/bin/bash"
+              - "-cx"
+              - "git clone --recursive --quiet ${repository} &&
+                 cd rust-code-analysis &&
+                 git -c advice.detachedHead=false checkout ${head_rev} &&
+                 cargo build --release &&
+                 cd target/release &&
+                 tar -zvcf /build.tar.gz rust-code-analysis"
```

Comment:
This is going to be different than the current directory structure. I guess the only two users are bugbug and microannotate for now, so we can live with the change.

Reply:
I can replicate the current one